pkgs/applications/editors/lite-xl/default.nix

```diff
@@ -15,13 +15,13 @@
 
 stdenv.mkDerivation rec {
   pname = "lite-xl";
-  version = "2.1.1";
+  version = "2.1.2";
 
   src = fetchFromGitHub {
     owner = "lite-xl";
     repo = "lite-xl";
     rev = "v${version}";
-    sha256 = "sha256-tZ9bCazs4ygNl5RKFNUtxboaMcG8a7mIz2FuiExX1d4=";
+    sha256 = "sha256-kl3vgBP+ExgZfdyOlqAEhUdR+kG8dAjyvVSm2I4e9yg=";
```

Comment:
```suggestion
    hash = "sha256-kl3vgBP+ExgZfdyOlqAEhUdR+kG8dAjyvVSm2I4e9yg=";
```

Reply:
Thanks, resolved in e17bbcca2e1bf27083433f01911a8d304dcd724e!